packages/lms/src/lib/components/navigation/Entity.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+    import { drawerStore } from '@skeletonlabs/skeleton';
+	import type { EntityMeta } from '@mollify/types';
+	export let entity: EntityMeta;
+	let open = false;
+    function drawerClose(): void {
+        drawerStore.close();
+    }
+    let typeOfEntity = entity.type;
+    
+    const entityIcons = {
+        'Assessment': 'Task',
+        'Lesson': 'Article',
+        'Module': 'Layers',
+        'Course': 'Book',
```

Comment:
I'm partial to `Book_5` myself.

Reply:
Hello @Anclagen, could you kindly review the rendering of icons? There may be an issue

Reply:
I'm not having any problems on my end but if your having issues with rendering icons it might get fixed when these are installed as a package in #162.